mirascope/integrations/langfuse/_with_langfuse.py

```diff
@@ -15,7 +18,7 @@
 )
 
 
-def custom_decorator(fn):
+def custom_decorator(fn: Callable) -> Callable:
```

Comment:
I think it's safe to assume that the custom decorator should leave the original callable unchanged?

Reply:
I think it is a function that returns a decorator, so I made the type stricter.